src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
 // write code here
+const tbody = document.querySelector('tbody');
+const rows = Array.from(tbody.rows);
```

Comment:
The `rows` array is initialized once and does not update when rows are added or removed. Consider updating the `rows` array inside the event listeners to ensure it reflects the current state of the table.